food-truck-api/src/main/java/food/truck/api/user/UserPreferences.java

```diff
@@ -9,5 +9,7 @@
     double acceptableRadius;
     double priceRating;
     int numRequested;
+    boolean active;
+    Set<String> truckIds;
```

Comment:
Since truckIDs themselves are `Long`s internally, why are we storing them here as `String`s? Just curious.